public/ge0.html

```diff
@@ -77,19 +98,25 @@
 
 <body>
   <div id="header">
-    <span id="name">${name}</span>
-    <span>
-      <a id="open" href="om:/${path}" class="link">
-        Open in Organic Maps
-      </a>
-      <a href="https://omaps.app/get" target="_blank" class="link">
-        Install Organic Maps
-      </a>
-      <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">
-        See on OpenStreetMap
+    <div id="container">
+      <a href="https://organicmaps.app/">
+        <img id="logo" src="/logo-icon.svg"/>
       </a>
-    </span>
+      <span id="name">${name}</span>
+      <div>
+        <a id="open" href="om:/${path}" class="link">
+          Open in Organic Maps
+        </a>
+        <a href="https://omaps.app/get" target="_blank" class="link">
+          Install Organic Maps
+        </a>
+        <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">
+          See on OpenStreetMap
+        </a>
+      </div>
```

Comment:
Please also add a link `geo:lat,lon?Z=zoom` with the easily copyable text "lat, lon" and ideally with a small copy-to-a-buffer button that copies just coordinates.

Reply:
I'm afraid I'm not quite sure what you mean,

do you mean a small button inside the 'See on OpenStreetMap' button that opens the `geo:` link?

Reply:
Is this what you mean? On pressing, it opens the `geo:` url with the pinned location.
<img width="1428" alt="Screenshot 2022-03-02 at 18 42 36" src="https://user-images.githubusercontent.com/26939824/156427273-20269602-da4d-4b18-9846-be816505e442.png">

Reply:
Yes, the second option. Would it be hard to also draw a 📋 symbol on the right of the coords inside a geo button, and add a copy-on-click JS code (with a popup that "lat, lon has been copied into clipboard")?

Reply:
It should be possible 👍 I might not be able to do it for a few days though unfortunately